src/components/Dashboard.tsx

```diff
@@ -0,0 +1,74 @@
+import Card from "src/components/Card";
+import Left from "src/components/DashboardLeft";
+import { useLocations } from "src/lib/hooks/useLocations";
+import { useEffect, useState } from "react";
+import type { PointTuple } from "leaflet";
+import type { Location } from "src/lib/database/location";
+
+/* load map without ssr due to lack of support with Leaflet */
+import dynamic from "next/dynamic";
+const MapWithNoSSR = dynamic(() => import("./DashboardMap"), {
+  ssr: false,
+}) as React.FC<{
+  locations: Location[];
+  selectedLocation: number;
+  mapCenter: PointTuple;
+}>;
+
+const paneStyle = {
+  width: "50%",
+  minHeight: "80vh",
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+};
+
+const mapCenter: PointTuple = [56.178516, 15.60261];
+
+const Dashboard: React.FC = () => {
+  const locations = useLocations("/api/v3/locations");

```

Comment:
locations are fetched in [_app.tsx](https://github.com/sensor-network/open-data-portal/blob/integrations/src/pages/_app.tsx) and passed through the PreferenceContext.
consider using that to reduce fetching